internal/proxy/impl.go

```diff
@@ -4407,6 +4407,35 @@ func (node *Proxy) CheckHealth(ctx context.Context, request *milvuspb.CheckHealt
 	}, nil
 }
 
+func (node *Proxy) RenameCollection(ctx context.Context, req *milvuspb.RenameCollectionRequest) (*commonpb.Status, error) {
+	ctx, sp := otel.Tracer(typeutil.ProxyRole).Start(ctx, "Proxy-RefreshPolicyInfoCache")
+	defer sp.End()
+
+	log := log.Ctx(ctx)
+	log.Debug("received rename collection request", zap.Any("request", req))
```

Comment:
- I think `Info` will be better here.
- Do not use `zap.Any` to print a struct, though maybe it's not large.